webapp/src/main/webapp/larva/assets/lib.js

```diff
@@ -396,9 +396,13 @@ function indentCompare(sources, result)
 {
 	var s = eval(sources);
 	
-	for (var i = 0; i < s.length; i++)
-		xmlFormat(s[i]);
-		
+	for (var i = 0; i < s.length; i++){
+		// if it is xml
```

Comment:
Since you're fetching `s[i]` 3 times, it might perform better if you assign the value first;
`var j = s[i];`